public_html/includes/js/develop/createForm_form.js

```diff
@@ -1531,8 +1531,9 @@ function addMetadataStandard(val){
 	}
 
 	var options    = metadataSchema;
-	var identifier = val.split(' : ')[1];
-	var standard   = val.split(' : ')[0];
+	val = val.replace(/\s+/g, '');
```

Comment:
i didn't remove the var. `val` is already defined as a parameter to the function. i'm reassigning `val` a new value (itself after modifications).